test/vagrant-service-manager/installer_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'test_helper'
+require "vagrant/util/downloader"
+
+module VagrantPlugins
```

Comment:
nice, that's what I meant with testing the downloader stuff standalone. No need for starting the VM. Having the capability to change single classes in isolation now is a great step forward!